Week3/homework/step2-7.js

```diff
@@ -21,3 +24,10 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+/** ******
+ * The answer in this link:
+https://www.w3schools.com/js/js_function_parameters.asp
+Exactly in these 2 titles:
```

Comment:
I am not too convinced with the explanation you've provided here. Firstly, I would have liked if you would have tried to explain it in your words rather than simply stating what is written in `w3schools`. Secondly, IMO, the statement `arguments are passed by value, objects are passed by reference` is not the most reasonable explanation for this because even objects can be function arguments. 🙂

Reply:
recently I am watching this:
https://youtu.be/Bv_5Zv5c-Ts
And I got the idea, The relationship between JavaScript and the memory.